conf/modules.config

```diff
@@ -665,12 +658,48 @@ process{
         ]
     }
 
-    withName: 'GENOTYPEGVCFS' {
-        ext.prefix       = {"${meta.id}.haplotypecaller"}
-        ext.when         = { params.tools && params.tools.split(',').contains('haplotypecaller') && params.joint_germline}
+    withName: 'GATK4_GENOMICSDBIMPORT' {
+        ext.prefix       = { meta.num_intervals > 1 ? meta.intervals_name : "joint_interval" }
+ext.when         = { params.tools && params.tools.split(',').contains('haplotypecaller') && params.joint_germline && !params.no_intervals}
+    }
+
+    withName: 'GATK4_GENOTYPEGVCFS' {
+        ext.prefix       = { meta.num_intervals > 1 ? meta.intervals_name: "joint_interval" }
```

Comment:
we had
`"${meta.id}.haplotypecaller"` for the prefix here before, no?

Reply:
And no more when statement for this one?

Reply:
either the merged and unmerged should have the same name so that regardless if 1 interval or a 100, the output file always has the same name